float64[] iq_meas
float64[] iq_meas_filt

float64[] alpha_f0
float64[] alpha_f1

float64[] K_d0_cal
float64[] K_d1_cal
float64[] K_t_cal
float64[] rotor_MoI_cal

float64[] sol_millis

string[] jnt_names
